nengo_spa/algebras/vtb_algebra.py

```diff
@@ -41,7 +41,15 @@ class VtbAlgebra(AbstractAlgebra):
 
     The VTB binding operation is neither associative nor commutative.
 
-    Publications with further information are forthcoming.
+    Fractional binding for this algebra is currently an experimental feature.
```

Comment:
I assume the `make_nondegenrate` method is part of this feature/also to be considered experimental?